server/sonar-server/src/main/java/org/sonar/server/plugins/ws/PluginWSCommons.java

```diff
@@ -142,9 +143,44 @@ public void writeArtifact(JsonWriter jsonWriter, Release release) {
     jsonWriter.endObject();
   }
 
+  /**
+   * Write an "update" object to the specified jsonwriter.
+   * <pre>
+   * "update": {
```

Comment:
this documentation does not seem to be up-to-date ("updates" has fields "releases" and "requires")